reference_pose_flag: 1
pose:
  position:
    x: 0.55686099076
    y: -0.296882237511
    z: -0.17712432692
  orientation:
    x: -0.00928787935556
    y: 0.989519808056
    z: -0.00726430566722
    w: -0.143914956223